source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -4,23 +4,19 @@ Run Kubernetes\* on |CL-ATTR|
 #############################
 
 This tutorial describes how to install, configure, and run the
-`Kubernetes container orchestration system`_ on |CL-ATTR| using different
-container engines and runtimes.
+`Kubernetes container orchestration system`_ on |CL-ATTR| using CRI+O and
```

Comment:
i am wonder if it would be nice to say that things might also work with runc and docker and nothing here means that it will not. just that this is a tutorial for kata. not all users of clear will need kata for their k8s cluster.